src/utils/helpers.js

```diff
@@ -79,3 +79,24 @@ export const sendMessageToRemoveBackwardsCompatibility = (currentVersion, minVer
         console.warn('You can now remove backwards-compatibility code from the project, as referenced in the stacktrace below.'); //eslint-disable-line no-console
     }
 };
+
+// Generates a RFC-4122 version 4 compliant globally unique identifier.
+export function generateId() {
```

Comment:
Can you make the `TestHelper.generateId` method in `test/test_helper.js` call this so that we're not duplicating the logic?

Reply:
Done

Reply:
are you going to be removing this helper?

Reply:
Not on this PR, but after this has been merged. I'll clean up that code - will delete `TestHelper.generateId()` and will just directly call `UtilsHelper.generateId()` for our test files.

(I initially thought to consider that in this PR but there are too many `TestHelper.generateId()` everywhere and I find it unnecessary for this PR.)